housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/conf/TunnelRoute.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.housekeeping.tool.vacuum.conf;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import javax.validation.Constraint;
+import javax.validation.OverridesAttribute;
+import javax.validation.Payload;
+import javax.validation.ReportAsSingleViolation;
+import javax.validation.constraints.Pattern;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Validates the annotated {@link String} is sequence of one or more {@link java.net.URI} separated by the literal
+ * {@code ->}, each of which contains an optional username and hostname only.
+ * <p>
+ * Refer to {@link TunnelRouteValidator} for more details about valid expressions.
+ * </p>
+ *
+ * @see TunnelRouteValidator
+ */
+@Documented
+@Constraint(validatedBy = { TunnelRouteValidator.class })
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER })
+@Retention(RUNTIME)
+@ReportAsSingleViolation
+@Pattern(regexp = "")
+public @interface TunnelRoute {
```

Comment:
I thought you were going to replace these classes with their versions from hcommon-ssh?

Reply:
This class is still in Circus Train so I'm assuming its needed

Reply:
What happens if you replace it with https://github.com/HotelsDotCom/hcommon-ssh/blob/master/src/main/java/com/hotels/hcommon/ssh/validation/constraint/TunnelRoute.java ? They are only very slightly different...